tests/cases/conformance/es2017/useObjectValuesAndEntries3.ts

```diff
@@ -0,0 +1,9 @@
+// @target: es6

```

Comment:
`// @lib: ess6,es2017.object` would make this file compile without errors, right?